examples/action/XDAI-bridge.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ts:action xmlns:ts="http://tokenscript.org/2019/05/tokenscript"
+           xmlns="http://www.w3.org/1999/xhtml"
+           xmlns:xml="http://www.w3.org/XML/1998/namespace"
+>
+    <ts:name>Transfer xDAI to DAI</ts:name>
+    <!-- because this action has xdai as input, it should be listed as
+         an action under XDAI's token view. -->
+    <ts:input>
+        <ts:token name="xdai">
+            <ts:ethereum network="100"/>
+        </ts:token>
+    </ts:input>
+    <!-- because this action has DAI as output, when a user decides to
+         create a new token, this action qallows DAI token to be chosen as
+         the new token to create -->
+    <ts:output>
+        <ts:token name="dai">
+            <ts:contract>
+                <ts:address network="1">0xdecafbad</ts:address>
```

Comment:
0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359

Reply:
Can you correct that and the `<to>` as well and remove the "WIP won't merge" tag?